src/main/java/com/kodilla/ecommercee/controller/CartController.java

```diff
@@ -39,6 +46,24 @@ public void deleteCart(@PathVariable Long cartId) {
         service.deleteCart(cartId);
     }
 
+    @GetMapping("/item/{itemId}")
```

Comment:
Sugestia - zamiast itemId może samo id -> wiadomo, że dotyczy item. Tylko jak poprawisz tu, to popraw wszędzie niżej, bo controller nie będzie chodził

Reply:
Jeśli zostawimy samo {itemId} to by adres wyszedł np. /v1/cart/5
Co by sugerowało szukanie cart o id 5 a nie itemu. Możemy tutaj zostawić to item